mono/metadata/boehm-gc.c

```diff
@@ -226,7 +228,9 @@ mono_gc_base_init (void)
 				}
 				continue;
 			} else if (g_str_has_prefix (opt, "toggleref-test")) {
+#if !HAVE_BDWGC_GC
 				register_test_toggleref_callback ();
```

Comment:
Only use appears to do anything with boehm when the MONO_GC_PARAMS environment variable includes (g_str_has_prefix (opt, "toggleref-test")) {